src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -4,7 +4,7 @@
   <Import Project="..\OrchardCore\OrchardCore.Application.Cms.Core.Targets\OrchardCore.Application.Cms.Core.Targets.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>$(CommonTargetFrameworks)</TargetFrameworks>
+    <TargetFramework>$(CommonTargetFrameworks)</TargetFramework>
```

Comment:
We'll have both .NET 8 and .NET 9 here (as we had .NET 7 and 8) when the latter comes out in November, so this needs to be kept the plural one.

Reply:
I know what you mean but there's an issue regarding TFMs. For multi-targeting I think we need to file an issue if we need to support LTS only or not, I remember last time we dropped .NET 7.0 support before the EOL

Reply:
Sorry, I don't understand.

Having the ability to always support the latest LTS and latest .NET version is more important than the build perhaps doing a bit of an extra work when copying localization files. We need to keep that capability.

Reply:
We might need to investigate if there's a better solution

Reply:
JT and I have spent days on this issue. The current code is the best we could do without breaking the builds randomly with concurrent file access issues, CF the issues filed by Scott Hansleman.

Reply:
Maybe try to anticipate the next release by installing net9.0 and changing the current target to handle 8 and 9, then repro the problem, and find a better solution. I am not very optimistic about how much time it would take for the potential gain.

Reply:
I have .NET 9.0 preview, I might try multi-targeting again